fronts-client/src/components/FrontsEdit/FrontContent.tsx

```diff
@@ -197,6 +248,8 @@ const mapStateToProps = () => {
     return {
       front: selectFront(state, { frontId: id }),
       alsoOn: selectAlsoOnFronts(state, { frontId: id }),
+      collectionsError: state.collections.error,
+      collectionsLastFetch: state.collections.lastFetch,
```

Comment:
We shouldn't access redux state directly. Instead, there are selectors in `asyncResourceBundle` for both of these pieces of state.